### ISSUE TITLE
Testing adding note for account manager chat

### ISSUE THREAD

**Issue by techievivek** [techievivek/webhook-testing#19]
- Affected Primary Email Address: jin@topdown.ca
- Concierge Chat: [Link](https://www.expensify.com.dev/concierge/#/chat/175)

## Research Checklist ✔️   
⚠️ Note - **all these steps** need to be completed before submitting a new issue to help our Concierge and Engineering teams investigate this further. Check out this [SO](https://stackoverflow.com/c/expensify/questions/14187) for research tips ⚠️ 
To check off a an item put an x in the parenthesis - `[x]` 

- [ ] **Searched GitHub for customer email in existing issues:** [if an OPEN duplicate exists, please add to that issue and do not continue creating this one. If a CLOSED issue exists, continue creating this issue and paste that link here]
- [ ] **Searched GitHub for issue customer is experiencing::** [if an OPEN duplicate exists, please add to that issue and do not continue creating this one. If a CLOSED issue exists, continue creating this issue and paste that link here]
- [x] **Admin email:** [Finance: jin@topdown.ca; Billing owner: skumar@topdown.ca (They have given permission to discuss billing with Jin]
- [x] **Policy IDs:** [3A14A8927878958F, 76E9C0C935F7434A, 8E80D5E531FA667E, D706D126185CF332, E35F615E58CAD393, F445DBC7F72962F4, FD12258A6A99872C, 4DD7DBA970F9DAFE
- [ ] **Report IDs:** [all affected report IDs]
- [ ] **Searched ZingTree:** [insert node link here] 
- [x] **Supportaled into customers account and took relevant screenshots** [Attached pdf of December bill]
- [ ] **Searched Slack for customer domain and email address:** [insert any links here]
- [ ] **Searched Slack for issue customer is experiencing:** [insert any links here]
- [ ] **Tried to reproduce in a test account** [If no, why not] 

## What is the customer trying to do?
User has stated on a meet greet call that they believe there is a discrepancy in the amounts between the policy breakdown box and the total of the bill. They are trying to view the policy breakdown on their billing receipt to allocate the cost of Expensify across departments.

## What is happening when they try do it? 
The sum of the amounts per policy don't add up to the total of the bill. The receipt total shows as $1404 before tax. The bottom box of the policy breakdown totals $961.21. 

## Additional Notes 
            
[@topdown.ca receipt.pdf](https://github.com/Expensify/Expensify/files/10189731/%40topdown.ca.receipt.pdf)

<img width="441" alt="image" src="https://user-images.githubusercontent.com/89941679/206577279-afc59548-36bd-440f-8d83-6a086a740300.png">

### Additional Notes: 
 &nbsp;Discrepancies of the billing breakdown&nbsp;
